src/components/Filter/Filter.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Filter from '../Filter/Filter';
+import { contactsMock } from '../../mocks/contactsMock';
+
+describe('Filter component', () => {
+  const onChangeMock = jest.fn();
+
+  const props = {
+    value: contactsMock[0].name,
```

Comment:
It is a bad practice. Use destructuring please

Reply:
Key value "value" in props changed to import name from contacts and destructuring was used.